content/10.diagnostics.md

```diff
@@ -180,11 +180,33 @@ These results suggest that immunity to SARS-CoV-1 is sustained for at least a ye
 The persistence of antibodies to SARS-CoV-2 remains under investigation.
 Circulating antibody titers to other coronaviruses have been reported to decline significantly after 1 year [@doi:10.1017/s0950268800048019].
 Autopsies of lymph nodes and spleens from severe acute COVID-19 patients showed a loss of T follicular helper cells and germinal centers that may explain some of the impaired development of antibody responses [@doi:10.1016/j.cell.2020.08.025].
-An early study (initially released on _medRxiv_ on February 25, 2020) presented a chemiluminescence immunoassay to a synthetic peptide derived from the amino acid sequence of the SARS_CoV-2 _S_ protein [@doi:10.1093/infdis/jiaa243].
+An early study (initially released on _medRxiv_ on February 25, 2020) presented a chemiluminescence immunoassay to a synthetic peptide derived from the amino acid sequence of the SARS-CoV-2 _S_ protein [@doi:10.1093/infdis/jiaa243].
 This method was highly specific to SARS-CoV-2 and detected IgM in 57.2% and IgG in 71.4% and 57.2% of sera samples from 276 confirmed COVID-19 patients.
 They reported that they could detect IgG within two days of the onset of fever and were not able to detect IgM any earlier, a pattern they compared to findings in MERS.
+Since then, several trials have reported the potential protective effect of antibodies in convalescent plasma obtained from recovered COVID-19 patients to treat critically ill COVID-19 patients [@doi:10.1172/jci138745; @doi:10.1001/jama.2020.4783; @doi:10.1056/NEJMoa2031893].
+
 <!--TODO: add https://www.nature.com/articles/s41591-020-0965-6#Sec9 and evalute reports of recurrence.-->
-The duration over which these antibodies persist remains unknown.
+Evidence to date suggests that sustained immunity to SARS-CoV-2 occurs for a period of at least 6-8 months [@doi:10.3201/eid2703.204543; @doi:10.1126/science.abf4063; @doi:10.1126/sciimmunol.abf8891].
+One study assessed sustained immunity using blood samples from 188 COVID-19 positive patients at various time points between 6 and 240 days post-symptom onset, whereby some patients were also assessed longitudinally [@doi:10.1126/science.abf4063]. 
+After 1 month, 98% of patients were seropositive for spike IgG. 
+Moreover, spike IgG titers were stable and heterogeneous among patients over a period of 6 to 8 months post-symptom onset, whereby 90% of subjects were seropositive at 6 months.
```

Comment:
```suggestion
Moreover, Spike IgG titers were stable and heterogeneous among patients over a period of 6 to 8 months post-symptom onset, with 90% of subjects seropositive at 6 months.
```
Is this saying that patients tended to follow their own, heterogeneous patterns over 6-8 months?  Also, it seems like most of their data was collected within 6 months, so we might want to be clear about the sample size for the 90%!